spec/controllers/idv/capture_doc_controller_spec.rb

```diff
@@ -104,7 +104,7 @@
         get :show, params: { step: 'capture_complete' }
 
         expect(@analytics).to have_received(:track_event).with(
-          Analytics::CAPTURE_DOC + ' visited', result
+          Analytics::CAPTURE_DOC + ' visited: capture_complete_show', result
```

Comment:
so this will create all-new events, breaking any old dashboards or queries we have. And we already have a `step` key inside the event properties.

Here's one I just pulled from the int logs, doesn't `properties.event_properties.step` give us something similar?

So the only data added here is the `action_name`? What if we just added `action_name` key to the `event_properties` hash?

```json
{
  "name": "Capture Doc visited",
  "properties": {
    "event_properties": {
      "step": "capture_complete",
      "step_count": 1
    },
    "user_id": "anonymous-uuid",
    "user_ip": "73.81.87.185",
    "hostname": "idp.int.identitysandbox.gov",
    "pid": 12953,
    "service_provider": "https://test.xms.hhs.gov/",
    "user_agent": "Mozilla/5.0 (iPhone; CPU iPhone OS 14_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.3 Mobile/15E148 Safari/604.1",
    "browser_name": "Mobile Safari",
    "browser_version": "14.0.3",
    "browser_platform_name": "iOS",
    "browser_platform_version": "14.4",
    "browser_device_name": "iPhone",
    "browser_device_type": "smartphone",
    "browser_bot": false
  },
  "time": "2021-02-09T15:33:14.001Z",
  "id": "42d38f62-c8b8-44e8-9f84-37fda6df57d5",
  "visitor_id": "134efbb7-293a-4cd6-b45f-7a8da8fecc72",
  "visit_id": "960c43e3-1234-4966-8df1-c39153e3afd3"
}
```

Reply:
I like this suggestion. Makes it easier to keep backwards compatibility, and any future queries can use `event_properties.action_name`

Reply:
Opted to duplicate the events with new ones with doc_auth names that match the rest of the IdV flow. See the main PR comment for details.